core/src/main/kotlin/no/strooped/view/screen/LobbyScreen.kt

```diff
@@ -0,0 +1,145 @@
+package no.strooped.view.screen
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.Screen
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.graphics.OrthographicCamera
+import com.badlogic.gdx.graphics.Texture
+import com.badlogic.gdx.graphics.g2d.BitmapFont
+import com.badlogic.gdx.graphics.g2d.SpriteBatch
+import com.badlogic.gdx.math.Rectangle
+import com.badlogic.gdx.math.Vector2
+import com.badlogic.gdx.math.Vector3
+import com.badlogic.gdx.scenes.scene2d.Stage
+import com.badlogic.gdx.scenes.scene2d.ui.Label
+import com.badlogic.gdx.utils.viewport.ScreenViewport
+import no.strooped.StroopedController
+import no.strooped.TextureSizes
+
+/**
+ * Uses https://otter.tech/an-mvc-guide-for-libgdx/ as inspiration
+ * */
+val FONT_SIZE_WELCOME_TEXT = TextureSizes.adjustedFontSize(3.0f)
+class LobbyScreen(val controller: StroopedController) : Screen {
+    var ui: Stage = Stage(ScreenViewport())
+
+    var cam: OrthographicCamera = OrthographicCamera()
+    var mouse: Vector3 = Vector3()
```

Comment:
Is  `mouse` used anywhere?

Reply:
No, this should be removed (copied from a different project)

Reply:
Done